modules/_labs/pagination/react/lib/Pages.tsx

```diff
@@ -0,0 +1,124 @@
+import {css, jsx} from '@emotion/core';
+import {IconButton} from '@workday/canvas-kit-react-button';
+import canvas from '@workday/canvas-kit-react-core';
+import _ from 'lodash';
+import React, {useRef} from 'react';
+
+import {PaginationAriaLabelsDefault} from './PaginationAriaLabels';
+
+/** @jsx jsx */
+interface PagesProps {
+  numPages: number;
+  currentPage: number;
+  onPageClick: (page: number) => void;
+  mobile: boolean;
+  ariaLabels: PaginationAriaLabelsDefault;
```

Comment:
I don't like prop objects like this. @anicholls @lychyi what do you think?

Pros:
- Namespacing similar properties
- Less props overall
- More processing - often confusing processing like the key filtering used somewhere

Cons:
- Documenting is harder. Documentation in Storybook will show a named interface as the type. Interplay will show a whole object (I think a JSON object)
- More props
- More clear what sub-properties are being used where

For the Tables API, we requested a flattening of the event handlers.

Reply:
The consensus was flattened attributes

Reply:
latest commit flattens the aria label props

Reply:
k. I'll update one of the labels. I noticed a misspelling of "ellipsis", but I'll resolve that conflict